spec/lib/tasks/youtube_spec.rb

```diff
@@ -1,24 +1,31 @@
-require 'spec_helper'
+# frozen_string_literal: true
 
 describe 'YouTube rake tasks' do
-  before(:all) do
-    @rake = Rake::Application.new
-    Rake.application = @rake
-    Rake.application.rake_require('tasks/youtube')
-    Rake::Task.define_task(:environment)
+  let(:rake) { Rake::Application.new }
+
+  before do
+    Rake.application = rake
+    $LOADED_FEATURES.reject! { |f| f == 'tasks/youtube.rake' }
```

Comment:
It took me a bit to figure out what was going on with this rejection line, but we shouldn't need that. This spec can be cleaned up quite a bit further while keeping Rubocop happy:
```ruby
describe 'YouTube rake tasks' do
  before(:all) { Rails.application.load_tasks }

  describe 'usasearch:youtube:refresh' do
    let(:task_name) { 'usasearch:youtube:refresh' }

    before { allow(YoutubeData).to receive(:refresh) }

    after { Rake::Task[task_name].reenable }

    it "has 'environment' as a prereq" do
      expect(Rake::Task[task_name].prerequisites).to include('environment')
    end

    it 'runs YoutubeData.refresh' do
      Rake::Task[task_name].invoke
      expect(YoutubeData).to have_received :refresh
    end
  end
end
```

Reply:
Changed.

Reply:
Backed out the change; doing this breaks our code coverage tool, and it incorrectly reports uncovered code in unrelated rake tasks. Precisely which ones depends on RSpec's ordering; any task's specs which ran before this one will show as not being covered by tests. It looks like the call to load_tasks resets the coverage counters for the tasks.

Reply:
In that case, let's simplify things and back out the changes to this file entirely, as no changes are necessary for this story.

Reply:
OK

Reply:
I left the # frozen_string_literal: true in, but backed out all of the other changes.